web-api/src/storage/elastic/client.rs

```diff
@@ -45,6 +46,14 @@ pub(crate) struct Config {
     #[serde(serialize_with = "serialize_redacted")]
     password: Secret<String>,
     index_name: String,
+
+    /// Request timeout in seconds.
+    #[serde(with = "serde_duration_as_secs", default = "default_timeout")]
+    timeout: Duration,
+}
+
+fn default_timeout() -> Duration {
+    Duration::from_secs(2)
```

Comment:
:wrench:  Just set the default in `impl Default` the `serde(default)` container attribute will grab it from there